unstructured/chunking/title.py

```diff
@@ -115,22 +120,37 @@ def chunk_by_title(
         metadata = first_element.metadata
         start_char = 0
         for element in section:
+            # -- concatenate all element text in section into `text` --
             if isinstance(element, Text):
+                # -- add a blank line between "squashed" elements --
                 text += "\n\n" if text else ""
```

Comment:
not related to this PR, but is this mistakenly prepending two blank lines to the start of every chunk? (if so, can just create the github issue to track)

Reply:
No. Text is initialized to `""` a couple lines above this, before iterating through the section elements. So the `else` part of this expression (`+= ""`) governs until after the text buffer has some text in it.